src/cubing/twisty/views/TwistyAlgViewer.ts

```diff
@@ -293,6 +301,7 @@ class AlgToDOMTree extends TraversalDownUp<DataDown, DataUp, DataUp> {
         earliestMoveIndex: dataDown.earliestMoveIndex + moveCount,
         twistyAlgViewer: dataDown.twistyAlgViewer,
         direction: dataDown.direction,
+        type: "setup",
```

Comment:
Open to alternate names for this and "execution" (unsure if there are technical terms for the different parts of a conjugate)